settings.gradle.kts

```diff
@@ -153,17 +154,9 @@ platform("ide") {
     subproject("tooling-api-builders")
 }
 
-// Native Platform
-platform("native") {
-    subproject("distributions-native")
-    subproject("platform-native")
-    subproject("language-native")
-    subproject("tooling-native")
-    subproject("testing-native")
-}
-
 // Software Platform
-platform("software") {
+val software = platform("software") {
+    uses(core)
```

Comment:
💅 How would we ensure these relationships stay up-to-date?  Is there any way to verify them against the actual dependency graph?

Reply:
Not yet, but that would be the plan at some point. For now, we would just keep using the existing process for making sure these intended relationships stay up-to-date (whatever that is). The only improvement that this PR makes is to document the intended relationships instead of them living as tribal knowledge.